db/db_test.cc

```diff
@@ -2546,18 +2550,18 @@ TEST_F(DBTest, PurgeInfoLogs) {
     // For mode (1), test DestroyDB() to delete all the logs under DB dir.
     // For mode (2), no info log file should have been put under DB dir.
     std::vector<std::string> db_files;
-    env_->GetChildren(dbname_, &db_files);
+    ASSERT_TRUE(env_->GetChildren(dbname_, &db_files).IsNotFound());
     for (std::string file : db_files) {
       ASSERT_TRUE(file.find("LOG") == std::string::npos);
     }
```

Comment:
This does not make sense to me.  If the GetChildren is returning NotFound, then the loop over db_files is never executed (nothing to loop on).

Reply:
I agree, the loop is not necessary.

Reply:
@sike-evolve Can you remove the redundant loop please?